src/goGetImport.ts

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+import vscode = require('vscode');
+import cp = require('child_process');
+import path = require('path');
+import fs = require('fs');
+import { getGoRuntimePath } from './goPath';
+import { getCurrentGoPath, getImportPath } from './util';
+
+export function goGetImport() {
+	const editor = vscode.window.activeTextEditor;
+	const selection = editor.selection;
+	const selectedText = editor.document.lineAt(selection.active.line).text;
+
+	const importPath = getImportPath(selectedText);
+
+	const goRuntimePath = getGoRuntimePath();
+	if (!goRuntimePath) {
+		return vscode.window.showErrorMessage('Could not locate Go binaries. Make sure you have Go installed');
+	}
+
+	fs.exists(path.join(getCurrentGoPath(), 'src', importPath), (exists) => {
+		if (exists) {
+			vscode.window.showInformationMessage(`Package ${importPath} already exists`);
+			return;
+		}
+
+		cp.execFile(goRuntimePath, ['get', '-u', importPath], (err, stdout, stderr) => {
+			if (stderr !== '') {
+				vscode.window.showErrorMessage(stderr);
+				return;
+			}
+
+			vscode.window.showInformationMessage(`Successfully fetched package ${importPath}`);
```

Comment:
What do you think of showing the progress/success/failure messages in the output pane just like the current command `Go: Install/Update Tools`

Reply:
I think that's a great idea.

Reply:
One thing I just thought of: `go get -u` doesn't output anything by default unless it fails. For this to be useful, we would have to run `go get -u -v`. We could also show the output pane only if the command fails. Thoughts?

Reply:
Agreed